app/assets/javascripts/table-sort.es6

```diff
@@ -0,0 +1,78 @@
+$(document).ready(function() {
+  if ($('table.sortable')) {
+    let table;
+    $('th.sort')
+      .wrapInner('<span class="sortable-column" title="sort this column"/>')
+      .each(function() {
+        const th = $(this),
+          thIndex = th.index();
+        let inverse = false;
+
+        th.click(() => {
```

Comment:
Why is the ES6 syntax for a function call being used here and not elsewhere?

Reply:
If you use ES6's `=>`, you don't have access to `this` inside the function.
And most other functions in this code are using `this`